rules/windows/lateral_movement_alternate_creds_pth.toml

```
@@ -0,0 +1,62 @@
+[metadata]
+creation_date = "2023/03/29"
+integration = ["windows", "endpoint"]
+maturity = "production"
+min_stack_comments = "The New Term rule type used in this rule was added in Elastic 8.4"
+min_stack_version = "8.4.0"
+updated_date = "2023/03/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may pass the hash using stolen password hashes to move laterally within an environment, 
+bypassing normal system access controls. Pass the hash (PtH) is a method of authenticating as a user 
+without having access to the user's cleartext password. 
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Potential Pass the Hash Attempt"
```

```suggestion
name = "Potential Pass-the-Hash (PtH) Attempt"
```
This one is optional, just to facilitate searchs